apps/tailwind-components/pages/input/RadioGroup.story.vue

```diff
@@ -2,12 +2,13 @@
   <InputTestContainer show-state v-slot="{ invalid, valid, disabled }">
```

Comment:
with the FormField story, should we not keep  these stories barebone to not confuse user suggesting that 'legend' is included in the input?

Reply:
That's a good point. I have another PR planned to fix the stories. I will add this to that story. It would be good to make all the stories behave in the same way.